features/scaffold-block.feature

```diff
@@ -126,6 +126,7 @@ Feature: WordPress block code scaffolding
     Then the {THEME_DIR}/blocks/fight-club.php file should exist
```

Comment:
2 tests are failing, need to be updated.

2 more tests need to verify if `style.css` file was created.